homeassistant/components/foscam/services.yaml

```diff
@@ -20,6 +20,8 @@ ptz:
             - "top_left"
             - "top_right"
             - "up"
+            - "sleep"
+            - "wake_up"
```

Comment:
Are sleep and wake up really categorized as pan and tilt movements by foscam? Shouldn't these be part of another service or select entity?

Reply:
Personally I can't answer that since my devices have no support but I understood them as sleep and wakeup positions, but now you mention it... they indeed sound like enable/disable camera options.